pyproject.toml

```diff
@@ -200,3 +200,11 @@ optional = true
 [tool.poetry.dependencies.pymongo]
 version = ">=3.8,<3.11"
 extras = [ "tls", "srv",]
+
+# uncomment once branch is updated and remove the last 3 lines
+#[tool.poetry.dev-dependencies.pytest-sanic]
+#git = "https://github.com/RasaHQ/pytest-sanic"
+#branch = "fix_signal_issue"
+[tool.poetry.dev-dependencies.pytest-sanic]
+git = "https://github.com/wochinge/pytest-sanic"
+branch = "fix-signal-issue"
```

Comment:
Can we switch to the official lib that Tobi mentioned here https://github.com/yunstanford/pytest-sanic/pull/63#issuecomment-1172429499?

Reply:
That's what I was trying to do on Friday afternoon before our discussion but it didn't work out, I can try again today.

Reply:
Gotcha, if that doesn't work out easily enough, let's just switch this URL to the RasaHQ one

Reply:
I left a comment on your PR to remove those 2 extra lines because it causes error, Tobi's change was sufficient.

Reply:
ah, got it - It was optimistic to hope that it would just work 😂

Reply:
Ok, the attempts this afternoon to use `sanic_testing` failed, every rewrite of `test_nlg()` failed :( So I have just changed the branch again in `pyproject.toml`.